docs/DeveloperGuide.md

```diff
@@ -804,3 +804,48 @@ testers are expected to do more *exploratory* testing.
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Effort**
+
+This section documents the effort in morphing AB3 to SWEe! .
+
+* General refactor of AB3 to SWEe!
+    * All instances of AddressBook and related words such as Person, Address, Email, Phone Number (and many more) had to be removed or refactored.
+    * This involved not only renaming the relevant methods and variables **throughout the project** but also entire classes, packages and documentation.
+    * Eg. The entire `Person` class in AB3 had to be morphed into `Flashcard` class with entirely new methods, parameters and variables.
+    * Eg. The `add` and `edit` commands (and their associated parsers) had to be rewritten as it was originally used to add/edit a `Person` but now they are used to add/edit a `Flashcard`.
+
+* `Quiz` and `Review` feature
+    * These 2 features are completely new in functionality relative to AB3 features.
+    * They involved much UI work as we had to design an entirely new screen. Our app now has 2 distinct screens as compared to the 1 screen of AB3.
+    * It was challenging to integrate these 2 features as they were not a conventional command to type into the commandbox:
+        * We had to handle keyboard input outside of the command box in these 2 features which is a functionality not present in AB3.
+        * We had to design a new pattern to activate these 2 commands and execute them (read Review implementation).
```

Comment:
Perhaps you could add a hyperlink here to the Review implementation section? Just a suggestion